client/send-text.cc

```diff
@@ -175,6 +185,24 @@ int main(int argc, char *argv[]) {
             }
         } while (run_forever && !interrupt_received);
     }
+    else if ( rot_vertical == true) { 
```

Comment:
potential fix in https://github.com/miloh/flaschen-taschen/commit/dce6aee716254e53fc6c2b65f0df6cb26141dea1

```
    if (scroll_delay_ms > 0) {
          if (!rot_vertical) {
                // regular 
          } else {
                // vertical
          }
    }
   else if (scroll_delay == 0) {
                          // don't scroll
             if (!rot_vertical) {
                // regular 
          } else {
                // vertical
          }
    }


```